modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -158,7 +159,10 @@ function AddFDSFilterModalContent({
 		setIsValidDateRange(isValid);
 	}, [from, to]);
 
-	const handleFilterSave = async () => {
+	const handleFilterSave = async (event: any) => {
+		setIsSubmitting(true);
+		event.preventDefault();
+
```

Comment:
We don't want to use explicit `any` type unless absolutely necessary.

Also, we like to have an empty line between invocations, so between line 163 and 164 we would like an empty line, for readability.

Reply:
Thanks for pointing it out, we don't really need to prevent the event at all :sweat_smile: